frontend/scss/organisms/_o-custom-tours-builder.scss

```diff
@@ -233,6 +236,33 @@
   }
 }
 
+.aic-ct-header__button--next {
```

Comment:
Please check if this should have 3px margin-left on smaller screens, before increasing to 9px.